Description: fix utf-8
 TODO: fixes a bad char
 .
 plantuml (7986-1) unstable; urgency=low
 .
   * New upstream version
Author: Enrique <kike+plantuml@eldemonionegro.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- plantuml-7986.orig/src/net/sourceforge/plantuml/graphic/DateEventUtils.java
+++ plantuml-7986/src/net/sourceforge/plantuml/graphic/DateEventUtils.java
@@ -57,7 +57,7 @@ public class DateEventUtils {
 			final List<String> asList = Arrays.asList("<u>June 29th, 1975",
 					"\"It was the first time in history that anyone had typed",
 					"a character on a keyboard and seen it show up on their",
-					"own computer’s screen right in front of them.\"", "\t\t\t\t\t\t\t\t\t\t<i>Steve Wozniak");
+					"own computer screen right in front of them.\"", "\t\t\t\t\t\t\t\t\t\t<i>Steve Wozniak");
 			return TextBlockUtils.mergeTB(textBlock, getComment(asList, color), HorizontalAlignment.LEFT);
 		}
 
